lighthouse/src/main.rs

```diff
@@ -138,6 +138,12 @@ fn main() {
                 .takes_value(true)
                 .global(true),
         )
+        .arg(
+            Arg::with_name("log-color")
+                .long("log-color")
```

Comment:
can we add an alias for the UK english fans? :grin:

should be as straight-forward as `.alias("log-colour")`